docker/python-verify/conda.Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements. See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership. The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE} as base_image
```

Comment:
The version number won't suffice, because we also need the TAG_PREFIX, or the TAG (formed from the apacheminificpp literal version number and tag prefix. e.g. apacheminifi:jammy-0.13.0). Not sure if any of those option would be more readable or cleaner. I did however remove some unused parameters from these dockerfiles in https://github.com/apache/nifi-minifi-cpp/pull/1504/commits/092da85dfa085b4f733b41633908242a7e7fec8b